src/core_editor/editor.rs

```diff
@@ -39,79 +40,91 @@ impl Editor {
     }
 
     pub(crate) fn run_edit_command(&mut self, command: &EditCommand) {
+        use EditCommand as EC;
```

Comment:
I'm not sure I see the point of this. I'd prefer to leave it EditCommand.

Reply:
Ok, no problem. In my opinion it's more noise than necessary, but I don't really care that much.

Reply:
Thanks. I'm really looking forward to playing with this.